dashboard/test/factories.rb

```diff
@@ -1,4 +1,17 @@
 FactoryGirl.define do
+  factory :plc_enrollment_unit_assignment, :class => 'Plc::EnrollmentUnitAssignment' do
```

Comment:
I can group them - I'd only ask that we all be more diligent about appending to factories at the end then. Rails scaffolding puts everything at the top